### ISSUE TITLE
Javascript files: how to set indentation to 4 spaces

### ISSUE THREAD

**Issue by Lobosque** [wsdjeg/SpaceVim#679]
I wonder how I change the indentation on javascript files from 2 to 4 spaces, as adding the configuration to  `~/.SpaceVim.d/init.vim` had no effect:

```
set tabstop=4
set shiftwidth=4
set expandtab
```

I did not find anything in the documentations so I believe this should be as simple as configuring a vanilla vim/neovim. Still, adding the settings above didn't work.
If I change the configuration after vim is up and running, it works as expected.

**Comment by Lobosque** [comment]
Just found out about
```g:spacevim_default_indent = 2```
Is there a section on the docs where all configuration options are documented? I did not find anything.

**Comment by wsdjeg** [comment]
:h SpaceVim

**Comment by gumbo2k** [comment]
https://spacevim.org/documentation/#code-indentation

TL;DR: 
Add `default_indent = 4` to the `[options]` block of your configuration in `~/.SpaceVim.d/init.toml`
If you need it just in one project you can add it to a local config file `.SpaceVim.d/init.toml` like this one:
```
[options]
    default_indent = 4
```

**Comment by wsdjeg** [comment]
Thanks for your tips :)
